scripts/fuzz_opt.py

```diff
@@ -952,7 +952,9 @@ def randomize_opt_flags():
             # longer working on the original test case but modified one, which
             # is likely to be called within wasm-reduce script itself, so
             # original.wasm and reduce.sh should not be overwritten.
-            if not given_wasm:
+            # Note that we can't do this if a.wasm doesn't exist, which can be
+            # the case if we failed to even generate the wasm.
+            if not given_wasm and os.path.exists('a.wasm'):
```

Comment:
It might be better to print a warning that wasm generation failed so we don't have a reduced test case?

Reply:
Good idea, added!